Dockerfile.template

```diff
@@ -51,7 +51,10 @@ RUN set -ex; \
 		| xargs -rt apt-mark manual; \
 	\
 	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
-	rm -rf /var/lib/apt/lists/*
+	rm -rf /var/lib/apt/lists/*; \
+    test "$(id -u www-data)" -eq 33; \
```

Comment:
You can't test this here, this is only executed during build time.
You have to do this in the entrypoint script which is executed on runtime. See e.g. here: https://github.com/docker-library/wordpress/blob/master/docker-entrypoint.sh#L27-L46